### PR TITLE
UI: [VAULT-19340] update test selectors for dashboard

### PR DESCRIPTION
**Pull request by kiannaquach**
**Description**
- Update test selectors for dashboard to be more general and dynamic

### CODE REVIEW

File: ui/app/templates/components/dashboard/quick-actions-card.hbs [hunk 1]
```diff
@@ -18,12 +18,12 @@
         @passObject={{true}}
         @objectKeys={{array "type" "version"}}
         class="is-marginless"
-        data-test-secrets-engines-select
+        data-test-search-select="secrets-engines"
```
Comment (hellobontempo) [inline ui/app/templates/components/dashboard/quick-actions-card.hbs:21]: I've also seen us leverage the test selector in the search select component with the `id` so here that'd be `[data-test-component="search-select"]#secret-engines-select`

No need to change, more of an FYI

**Review by hellobontempo — APPROVED** [review]
Nice work! 🚀

File: ui/tests/helpers/components/dashboard/dashboard-selectors.js
Comment (hellobontempo) [inline ui/tests/helpers/components/dashboard/dashboard-selectors.js:1]: Love how easy these are to find! 🤩

File: ui/app/templates/components/dashboard/quick-actions-card.hbs (hunk 1 above)
Comment (kiannaquach) [reply ui/app/templates/components/dashboard/quick-actions-card.hbs:21]: good suggestion! will keep that in mind next time!

**Pull request merged**